modules/contact-form/grunion-editor-view.php

```diff
@@ -0,0 +1,259 @@
+		</label>
+		<div class='clear-form'></div>
+	</div>
+</script>
+
+<script type="text/html" id="tmpl-grunion-field-checkbox-multiple">
+	<div>
+		<label class='grunion-field-label'>{{ data.label }}<# if ( data.required ) print( " <span>" + data.required + "</span>" ) #></label>
+		<# _.each( data.options, function( option ) { #>
+			<label class='grunion-checkbox-multiple-label checkbox-multiple'>
+				<input type='checkbox' name='{{ data.id }}[]' value='{{ option }}' class="{{ data.class }}" <# if ( option === data.value || _.contains( data.value, option ) ) print( "checked='checked'" ) #> />
+				<span>{{ option }}</span>
+			</label>
+		<# }); #>
+		<div class='clear-form'></div>
+	</div>
+</script>
+
+<script type="text/html" id="tmpl-grunion-field-select">
+	<div>
+		<label for='{{ data.id }}' class='grunion-field-label select'>{{ data.label }}<# if ( data.required ) print( " <span>" + data.required + "</span>" ) #></label>
+		<select name='{{ data.id }}' id='{{ data.id }}' class="{{ data.class }}">
+			<# _.each( data.options, function( option ) { #>
+				<option <# if ( option === data.value ) print( "selected='selected'" ) #>>{{ option }}</option>
+			<# }); #>
+		</select>
+	</div>
+</script>
+
+<script type="text/html" id="tmpl-grunion-field-date">
+	<div>
+		<label for='{{ data.id }}' class='grunion-field-label {{ data.type }}'>{{ data.label }}<# if ( data.required ) print( " <span>" + data.required + "</span>" ) #></label>
+		<input type='date' name='{{ data.id }}' id='{{ data.id }}' value='{{ data.value }}' class="{{ data.class }}" />
+	</div>
+</script>
+
+<script type="text/html" id="tmpl-grunion-field-text">
+	<div>
+		<label for='{{ data.id }}' class='grunion-field-label {{ data.type }}'>{{ data.label }}<# if ( data.required ) print( " <span>" + data.required + "</span>" ) #></label>
+		<input type='text' name='{{ data.id }}' id='{{ data.id }}' value='{{ data.value }}' class='{{ data.class }}' placeholder='{{ data.placeholder }}' />
+	</div>
+</script>
+
+
+<script type="text/html" id="tmpl-grunion-field-edit">
+	<div class="card is-compact grunion-field-edit grunion-field-{{ data.type }}" aria-label="<?php esc_attr_e( 'Form Field', 'jetpack' ); ?>">
+		<label class="grunion-name">
+			<span><?php esc_html_e( 'Field Label', 'jetpack' ); ?></span>
+			<input type="text" name="label" placeholder="<?php esc_attr_e( 'Label', 'jetpack' ); ?>" value="{{ data.label }}"/>
+		</label>
+
+		<?php
+		$grunion_field_types = array(
+			'text'              => __( 'Text', 'jetpack' ),
+			'name'              => __( 'Name', 'jetpack' ),
+			'email'             => __( 'Email', 'jetpack' ),
+			'url'               => __( 'Website', 'jetpack' ),
+			'textarea'          => __( 'Textarea', 'jetpack' ),
+			'checkbox'          => __( 'Checkbox', 'jetpack' ),
+			'checkbox-multiple' => __( 'Checkbox with Multiple Items', 'jetpack' ),
+			'select'            => __( 'Drop down', 'jetpack' ),
+			'radio'             => __( 'Radio', 'jetpack' ),
```

Comment:
Every one of these translatable strings could really use context. Some of them need it badly, like `Drop down` can be translated at least three different ways without knowing what it means. If we already have strings like these, I would propose just adding translator comments to avoid creating new strings:

```
/* translators: label name to choose a dropdown field for a contact form */
'select' => __( 'Drop down', 'jetpack' ),
```

Also, shouldn't it be `Dropdown`?